flair/datasets/sequence_labeling.py

```diff
@@ -646,7 +646,9 @@ def _convert_lines_to_sentence(
 
             # otherwise, this line is a token. parse and add to sentence
             token = self._parse_token(line, word_level_tag_columns, token)
-            sentence.add_token(token)
+            tokens.append(token)
+
+        sentence: Sentence = Sentence(text=tokens)
```

Comment:
This looks much cleaner than creating an empty sentence and adding tokens. I wonder if this means we can retire the `add_token` method entirely in a separate PR?

Reply:
Looking at the usage after this PR, we still have the following usages:

* Tests, which can be rewritten
* `UniversalDependenciesDataset` which works the same way as the ColumnDataset worked before
* Sentence - constructor, which I would prefer to keep.

We can surely make the `add_token` method private

Reply:
I added a proposal to this PR